stuff/template/form-template.html

```diff
@@ -119,27 +119,54 @@
      height: 320px;
   }
   .nav-item {}  /* tagging class */
+
+    #editdiv {
+    float: right;
+      height: 40px;
+   margin-top: 8px;
+      display: inline-block;
+  }
+  .editmoder {
+    text-align: right;
+  }
+
+  .edit-toggle{
+    cursor:pointer;
+  }
+
   </style>
   <script>
-  var form_is_enabled;
-  function enable_form(enable_action) {
-      var elements = document.querySelectorAll("#compform input:not(.nav-item),textarea");
-      for (var i = 0; i < elements.length; ++i) {
-	  elements[i].disabled = !enable_action;
-	  if (enable_action) {
-	      elements[i].style["background-color"] = 'white';
-	  } else {
-	      elements[i].style["background-color"] = '#eeeeee';
-	  }
-      }
-      form_is_enabled = enable_action;
-      edit_toggle = document.getElementById("edit-toggle");
-      if (edit_toggle) {
-          edit_toggle.src=form_is_enabled
-	   ? '/static/edit-pen.png'
-	   : '/static/non-edit-pen.png';
+
+    var form_is_enabled;
+
+    function enable_form(enable_action) {
+        var elements = document.querySelectorAll("#compform input:not(.nav-item),textarea");
+        for (var i = 0; i < elements.length; ++i) {
+          elements[i].disabled = !enable_action;
+          var tx = document.getElementById("edittext")
+
+
+      if (enable_action) {
+          elements[i].style["background-color"] = "white";
+          tx.textContent = "(on)"
```

Comment:
Same here: for every input element you set the global text-field.
You can do that simply once at the beginning of the function; something like
```
   document.getElementById("edittext").textContent = enable_action ? "(on)" : "(off)"
```